tests/pyccel/test_pyccel.py

```diff
@@ -1083,7 +1083,6 @@ def test_function(language):
             language = language, output_dtype=[str]+[int]*8 )
 
 #------------------------------------------------------------------------------
-@pytest.mark.xdist_incompatible
```

Comment:
Why was this mark removed? It should be here when more than one test translates the same function (in this case `scripts/decorators_inline.py`) so we don't get a race condition where one file tries to translate a shared library.

(Although in this case you won't have an issue as the other function using this file has the mark so they will not be run simultaneously in any case)

Reply:
[c2e23bc](https://github.com/pyccel/pyccel/pull/1728/commits/c2e23bc1bcb085e9d0d52aafecf258474e0a1313)